src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/email/Mailbox.java

```diff
@@ -45,11 +45,9 @@ public void deliverEmail(Email email, boolean isUnread) {
     this.emails.put(email, Set.of(isUnread ? Label.UNREAD : Label.READ));
   }
 
-  public List<Email> getEmailsWithLabel(Label... labels){
-    return emails.entrySet().stream()
-        .filter(entry -> entry.getValue().containsAll(Set.of(labels)))
-        .map(Map.Entry::getKey)
-        .toList();
+  public List<Email> getEmailsWithLabel(Label... labels) {
+    return emails.entrySet().stream().filter(entry -> entry.getValue().containsAll(Set.of(labels)))
```

Comment:
correct format ?

Reply:
Yup! @jens1o messed it up again with his settings :)